lib/user-interface/react/src/components/model-management/create-model/CreateModelModal.tsx

```diff
@@ -141,6 +145,8 @@ export function CreateModelModal (props: CreateModelModalProps) : ReactElement {
         }
     }
 
+    const requiredFields = [['modelId', 'modelName'], [], [], [], []];
```

Comment:
We can make streaming optional by setting it to false in the api model under the hood, that might simplify things, but I believe we also need modelType here too. as far as i can tell, required fields are modelId, modelName, streaming (which is easy to remove), and modelType (textgen/embedding)

Reply:
From a UI perspective there is no need to make this optional since it is a toggle. I can default it to false instead of true if you like

Reply:
We also already default the picker to Textgen for model type.. So there isn't a way that it can be submitted without a type since it doesn't have a clear option

Reply:
Ah, sounds good- backend brain is not factoring in frontend reality :D